### PR TITLE
PS-4573 5.7 Patch breaks creates .cnf file parsing problem

### PR DESCRIPTION
**Pull request by patrickbirch**
Added text to the yum_repo.

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for comments or approval.

File: doc/source/installation/yum_repo.rst [hunk 1]
```diff
@@ -78,13 +78,16 @@ Installing |Percona Server| from Percona ``yum`` repository
 
 1. Install the Percona repository 
    
-   You can install Percona yum repository by running the following command as a ``root`` user or with sudo:
+By default, Percona Server installs and updates creates a mysqld.cnf as
```
Comment (tplavcic) [inline doc/source/installation/yum_repo.rst]: a bit strange `installs and updates creates a mysqld.cnf` shouldn't it be `installs and updates a mysqld.cnf` or something similar?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: doc/source/installation/yum_repo.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst]: Rewrote the paragraph.

**Review by tplavcic — APPROVED** [review]
LGTM

**Pull request merged**